File: 1013 数素数 (20 分).cpp
#include<iostream>
#include<cmath>
using namespace std;
int main()
{
    int i,j;
    int M,N;
    int count=0;
    int showNUM=0;
    cin>>M>>N;
    for(i=2;i<=104730;i++)
    {
        for(j=2;j<=sqrt(i);j++)
        {
            if(i%j==0)
                break;
        }
        if(j>sqrt(i))
        {
            count++;
            if (count>N)
                break;
            if(count>=M)
            {
                showNUM++;
                if(showNUM%10==0 || count==N)
                    cout<<i<<endl;
                else
                    cout<<i<<" ";
            }
        }
        
    }
}   

File: 1013 数素数 (20 分).py
M,N=(map(int,input().split()))
count=0 #素数的个数
showNUM=0 #已经输出素数的个数，便于换行
for i in range(2,10001):  
    for j in range(2,i):
        if (i % j) == 0:
            break
    else:
        count=count+1
        if count>N:
            break
        if count>=M:
            showNUM=showNUM+1
            if showNUM%10==0 or count==N:               
                print (i)
            else:
                print (i,end=" ")
            